libraries/AP_GPS/AP_GPS.cpp

```diff
@@ -504,6 +504,14 @@ bool AP_GPS::vertical_accuracy(uint8_t instance, float &vacc) const
     return false;
 }
 
+bool AP_GPS::time_accuracy(uint8_t instance, float &t_acc) const
+{
+    if (state[instance].have_time_accuracy) {
+        t_acc = state[instance].time_accuracy * 1.0E-9;
+        return true;
+    }
+    return false;
+}
```

Comment:
This might be useful - but not really required in this PR?  It's currently unused.  Including this costs us even if we don't have OpenDroneID enabled.

Srsly - if we have a GPS fix then this is <1s.  Even if we lose GPS this will stay accurate to within 1s for a long time.

Reply:
Agree it could be removed. The timestamp_accuracy perplexed me a bit in the standard. Given it requires GPS to takeoff and if you have gps timestamps the accuracy is measured in nano-seconds...

 Where we set timestamp accuracy we could just do something like, 

If we have GPS the timestamp accuracy is less than 0.1 seconds.

If there is no GPS then we eventually we need decrease the accuracy up to MAV_ODID_TIME_ACC_1_5_SECOND. BUt as you said that could take a long while. But how long?

Reply:
removed